src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -457,4 +457,19 @@
   <data name="MissingKeepAlive" xml:space="preserve">
     <value>Missing argument for '/keepalive' option.</value>
   </data>
+  <data name="AnalyzerTotalExecutionTime" xml:space="preserve">
+    <value>Total analyzer execution time: {0} seconds.</value>
+  </data>
+  <data name="MultithreadedAnalyzerExecutionNote" xml:space="preserve">
+    <value>NOTE: The actual wall clock time is likely lesser than the above execution time due to multithreaded analyzer execution.</value>
```

Comment:
"Lesser" is not correct. Looks like the website lost the comments, but I have it in my email:
There were a few comments, but ultimately John suggested:
>Elapsed time may be less than analyzer execution time because analyzers can run concurrently.

I like that.